server/controller/todoController.ts

```diff
@@ -0,0 +1,82 @@
+import { Request, Response } from 'express';
+import todoService from '../service/todoService';
+import { sendSuccessResponse, sendErrorResponse } from './response';
+
+class TodoController {
+  async createDefaultTodo(req: Request, res: Response) {
+    try {
+      const userId = req.params.userId;
```

Comment:
can you add a validation to check if the userId is provided , if not throw an error